public/src/main/java/com/revenuecat/purchases/models/SubscriptionOptions.kt

```diff
@@ -70,8 +70,9 @@ class SubscriptionOptions(
         }.minByOrNull { it.second }?.first
     }
 
-    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    internal fun billingPeriodToDays(period: Period): Int {
+    // TODO make internal
```

Comment:
Do we still need to make this internal?

Reply:
It's used in the public module (SubscriptionOptions.kt) so for now it needs to be public until we move `public` into `purchases`